src/App.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+const todoList = [
+  {
+    id: 1,
+    title: "Complete coding assignment",
+  },
+  {
+    id: 2,
+    title: "Go grocery shopping",
+  },
+  {
+    id: 3,
+    title: "Study for React class",
+  },
+];
+
+function App() {
+  return (
+    <div>
+      <h1>Todo List</h1>
+      <ul>
+        {todoList.map(function(item) {
+          return <li key={item.id}>{item.title}</li>;
+        })}
```

Comment:
This is bit of a nitpick. Everything looks good here :)
But, how about try an anonymous function and implicit return here. It will make the code a bit cleaner.
So something like this. Does this make sense?

```suggestion
        {todoList.map(item => <li key={item.id}>{item.title}</li>)}
```

Reply:
It does make sense. Thank you Kasozi.